### DISCUSSION TITLE
Migrations not getting applied to database on deployment

### DISCUSSION THREAD

**Discussion by ZiadNOsman** [symfony/symfony#47706] [Q&A]
This issue only showed up after 
1. upgrading to Symfony 6.1 from Symfony 6.0
2. upgrading to php 8.1 from php 8.0
3. downgrading to maker-bundle v1.43.0 from v1.45.0

not sure which of these changes caused the issue, or if they even have anything to do with it.

prior to these changes, after committing a new migration from local, it would automatically be applied to the production database.

I checked the doctrine_migration_versions table in my production database, and the last migration applied is not the migration I newly pushed, which confirms that the new migrations are not getting applied to my database.

If anyone can give me some pointers on what could be the issue, it would be a life saver.

Note: the migrations are working just as expected on local, and the new migration gets detected locally when I run doctrine:migration:migrate

**Comment by ghost** [comment]
Don't you have a staging server with the same configuration as a production? It's unsafe to deploy from local to production right away.

**Reply by ZiadNOsman** [reply]
We do have staging and prod.
I solved the issue and it was caused by something else entirely.
docker-entrypoint.sh has a line that searches for a line starting with "DATABASE_URL" in the .env folder. 
`grep -q ^DATABASE_URL= .env`
However our .env file started with a space before each line, so even though we had DATABASE_URL defined, it was not picked up by the regular expression
